packages/block-library/src/image/index.php

```diff
@@ -223,21 +219,26 @@ function block_core_image_render_lightbox( $block_content, $block ) {
 	$img = null;
 	preg_match( '/<img[^>]+>/', $body_content, $img );
 
-	$button =
-		$img[0]
-		. '<button
-			class="lightbox-trigger"
```

Comment:
maybe worth checking if anything in Gutenberg or other plugins references this class before removing it.